sipservice/src/main/java/net/gotev/sipservice/SipCall.java

```diff
@@ -274,14 +274,15 @@ public void sendBusyHereToIncomingCall() {
         }
     }
 
-    public void declineIncomingCall() {
+    public void declineIncomingCall() throws Exception{
```

Comment:
Check on unused methods..

Reply:
All changes will be accommodated in AND-3730 PR.